test/files/neg/t5887.scala

```diff
@@ -3,9 +3,28 @@ trait TheOldCollegeTry {
 
   // was: value isDefinedAt is not a member of Int
   // now: required: PartialFunction[Throwable,?]
```

Comment:
stale

Reply:
Dale! oh right, as Lukas hinted, it expects a function but will leverage a PF. I'll update the description accordingly.